tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsLatestSpecific.g.cs

```diff
@@ -1256,7 +1352,57 @@ public class PackageVersionsLatestSpecific
 #if NET8_0
                 new object[] { "2.0.0" },
                 new object[] { "2.2.10" },
-                new object[] { "3.5.2" },
```

Comment:
Oops didn't catch this version bump to `3.6.0`, hopefully okay?

Reply:
> Oops didn't catch this version bump to 3.6.0, hopefully okay?

Yeah, that should be fine, we merged support for 3.6.0 already. Though why aren't the _other_ ones bumped to 3.6.0 too 🤔

Reply:
Yes, this is weird, I wonder if is something to do with running the generator on mac or something...

Reply:
Updated the other ones - I was on a different branch so I deleted a previous commit which is why the others weren't updated either